### DISCUSSION TITLE
Mplayer/mpv has finished unexpectedly exit code: 1

### DISCUSSION THREAD

**Discussion by cod3mas0n** [smplayer-dev/smplayer#284] [General]
hi smplayer crashes every one 
i try every thing but its still crashed
" Mplayer/mpv has finished unexpectedly exit code: 1 "

**Comment by cod3mas0n** [comment]
this is log :



/usr/bin/mplayer -noquiet -slave -identify -nofs -sub-fuzziness 1 -vo xv,x11, -nodr -double -nomouseinput -input nodefault-bindings:conf=/dev/null -nokeepaspect -wid 9 -monitorpixelaspect 1 -osdlevel 0 -subfont-osd-scale 3 -progbar-align 80 -ass -embeddedfonts -ass-line-spacing 0 -ass-font-scale 1 -noflip-hebrew -ass-styles /home/zed/.config/smplayer/styles.ass -subcp ISO-8859-1 -subpos 100 -volume 50 -vf-add screenshot=/home/zed/Pictures/smplayer_screenshots/shot -noslices -af-add scaletempo -softvol -softvol-max 110 /run/media/zed/Learn & Movie/Learn/LPIC 1/101/01 - 101 - شناسایی و تنظیم سخت افزارها.mp4

MPlayer SVN-r38302-11 (C) 2000-2021 MPlayer Team
do_connect: could not connect to socket
connect: No such file or directory
Failed to open LIRC support. You will not be able to use your remote control.

Playing /run/media/zed/Learn & Movie/Learn/LPIC 1/101/01 - 101 - شناسایی و تنظیم سخت افزارها.mp4.
libavformat version 58.76.100 (external)
libavformat file format detected.
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x7f86623cf360]Protocol name not provided, cannot determine if input is local or a network protocol, buffers and access patterns cannot be configured optimally without knowing the protocol
ID_VIDEO_ID=0
[lavf] stream 0: video (h264), -vid 0
ID_AUDIO_ID=0
[lavf] stream 1: audio (aac), -aid 0, -alang und
VIDEO:  [H264]  1366x768  24bpp  9.000 fps  307.8 kbps (37.6 kbyte/s)
X11 error: BadMatch (invalid parameter attributes)
Opening video filter: [ass auto=1]
[ass] auto-open
Opening video filter: [screenshot=/home/zed/Pictures/smplayer_screenshots/shot]
[ass] Shaper: FriBidi 1.0.10 (SIMPLE) HarfBuzz-ng 2.8.2 (COMPLEX)
[ass] Using font provider fontconfig
==========================================================================
Opening video decoder: [ffmpeg] FFmpeg's libavcodec codec family
libavcodec version 58.134.100 (external)
Selected video codec: [ffh264] vfm: ffmpeg (FFmpeg H.264)
==========================================================================
ID_VIDEO_CODEC=ffh264
Clip info:
 major_brand: mp42
ID_CLIP_INFO_NAME0=major_brand
ID_CLIP_INFO_VALUE0=mp42
 minor_version: 0
ID_CLIP_INFO_NAME1=minor_version
ID_CLIP_INFO_VALUE1=0
 compatible_brands: mp42isomavc1
ID_CLIP_INFO_NAME2=compatible_brands
ID_CLIP_INFO_VALUE2=mp42isomavc1
 creation_time: 2015-02-11T14:33:17.000000Z
ID_CLIP_INFO_NAME3=creation_time
ID_CLIP_INFO_VALUE3=2015-02-11T14:33:17.000000Z
 encoder: HandBrake 0.9.9 2013052900
ID_CLIP_INFO_NAME4=encoder
ID_CLIP_INFO_VALUE4=HandBrake 0.9.9 2013052900
ID_CLIP_INFO_N=5
Load subtitles in /run/media/zed/Learn & Movie/Learn/LPIC 1/101/
ID_FILENAME=/run/media/zed/Learn & Movie/Learn/LPIC 1/101/01 - 101 - شناسایی و تنظیم سخت افزارها.mp4
ID_DEMUXER=lavfpref
ID_VIDEO_FORMAT=H264
ID_VIDEO_BITRATE=307808
ID_VIDEO_WIDTH=1366
ID_VIDEO_HEIGHT=768
ID_VIDEO_FPS=9.000
ID_VIDEO_ASPECT=0.0000
ID_AUDIO_FORMAT=MP4A
ID_AUDIO_BITRATE=139112
ID_AUDIO_RATE=48000
ID_AUDIO_NCH=1
ID_START_TIME=0.00
ID_LENGTH=1857.11
ID_SEEKABLE=1
ID_CHAPTERS=0
==========================================================================
Opening audio decoder: [ffmpeg] FFmpeg/libavcodec audio decoders
AUDIO: 48000 Hz, 1 ch, floatle, 139.1 kbit/9.06% (ratio: 17389->192000)
ID_AUDIO_BITRATE=139112
ID_AUDIO_RATE=48000
ID_AUDIO_NCH=1
Selected audio codec: [ffaac] afm: ffmpeg (FFmpeg AAC (MPEG-2/MPEG-4 Audio))
==========================================================================
AO: [pulse] 48000Hz 1ch floatle (4 bytes per sample)
ID_AUDIO_CODEC=ffaac
[Mixer] No hardware mixing, inserting volume filter.
Starting playback...
Movie-Aspect is 1.78:1 - prescaling to correct movie aspect.
ID_VIDEO_ASPECT=1.7786
[swscaler @ 0x7f8662494b20]bicubic scaler, from yuv420p to rgb24 using MMXEXT
[swscaler @ 0x7f8662494b20]using unscaled yuv420p -> rgb24 special converter
VO: [xv] 1366x768 => 1366x768 Planar YV12 
X11 error: BadWindow (invalid Window parameter)
X11 error: BadWindow (invalid Window parameter)
X11 error: BadDrawable (invalid Pixmap or Window parameter)
X11 error: BadWindow (invalid Window parameter)
X11 error: BadDrawable (invalid Pixmap or Window parameter)
X11 error: BadWindow (invalid Window parameter)
X11 error: BadDrawable (invalid Pixmap or Window parameter)
X11 error: BadWindow (invalid Window parameter)

X11 error: BadDrawable (invalid Pixmap or Window parameter)
X11 error: BadDrawable (invalid Pixmap or Window parameter)
X11 error: BadWindow (invalid Window parameter)


MPlayer interrupted by signal 8 in module: filter video
ID_SIGNAL=8
- MPlayer crashed by bad usage of CPU/FPU/RAM.
  Recompile MPlayer with --enable-debug and make a 'gdb' backtrace and
  disassembly. Details in DOCS/HTML/en/bugreports_what.html#bugreports_crash.
- MPlayer crashed. This shouldn't happen.
  It can be a bug in the MPlayer code _or_ in your drivers _or_ in your
  gcc version. If you think it's MPlayer's fault, please read
  DOCS/HTML/en/bugreports.html and follow the instructions there. We can't and
  won't help unless you provide this information when reporting a possible bug.

**Reply by smplayer-dev** [reply]
It crashes with a video filter. Try disabling all video filters.
Or use mpv instead.
